yos/list.go

```diff
@@ -0,0 +1,87 @@
+package yos
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+type FilePathInfo struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type FilePathInfo should have comment or be unexported](https://app.codacy.com/app/an9an63/gut/pullRequest?prid=4666961)